packages/opentelemetry-exporter-prometheus/test/prometheus.test.ts

```diff
@@ -68,17 +68,13 @@ describe('PrometheusExporter', () => {
         }
       );
     });
-
-    it('should not start the server by default', () => {
```

Comment:
I think this test is still relevant , if user doesn't provide anything (when using in pure js for example) the server should not start right ?

Reply:
> I would argue that boolean parameter should be ever required. By default anything what is boolean and is not defined is simply `false`, we should not change it, In js if user doesn't define config we will have "an object undefined exception", I don't see any reason why we should have it. And just writing everywhere
> 
> ```
> startServer: false
> ```
> 
> doesn't look very intuitive why would you even want that

@legendecas what do you think about inverting the name to `preventStartServer` or similar?